bgpd/bgp_evpn.c

```diff
@@ -2606,7 +2609,7 @@ static int install_evpn_route_entry(struct bgp *bgp, struct bgpevpn *vpn,
 			break;
 
 	if (!pi)
-		bgp_create_evpn_bgp_path_info(parent_pi, rn);
+		pi = bgp_create_evpn_bgp_path_info(parent_pi, rn);
```

Comment:
I see here it's not used as warning says 
![Screenshot from 2019-10-17 08-09-22](https://user-images.githubusercontent.com/3352707/66979231-7984bd80-f0b5-11e9-98c6-3bde723775a0.png)

Reply:
As per the latest code this variable did get used by passing to this function at line 2575.

	bgp_aggregate_increment(bgp_vrf, &rn->p, pi, afi, safi);

and also at line 2581

	vpn_leak_from_vrf_update(bgp_get_default(), bgp_vrf, pi);

Reply:
Yes, but it's a different function.

Reply:
I mean `install_evpn_route_entry()` and `install_evpn_route_entry_in_vrf()`